Cryptocurrency/multichain_gas_widget.5s.rb

```diff
@@ -4,16 +4,17 @@
 # <bitbar.version>v1.0</bitbar.version>
 # <bitbar.author>Masumi Kawasaki</bitbar.author>
 # <bitbar.author.github>geeknees</bitbar.author.github>
-# <bitbar.desc>Widget for monitoring Ethereum Gas Fees from https://etherscan.io/</bitbar.desc>
+# <bitbar.desc>Widget for monitoring Gas Fees from https://etherscan.io/</bitbar.desc>
 # <bitbar.dependencies>ruby</bitbar.dependencies>
 # <bitbar.image>https://raw.githubusercontent.com/geeknees/xbar-plugins/main/eth_gas_widget/screenshot.png</bitbar.image>
 # <bitbar.abouturl>https://github.com/geeknees/xbar-plugins</bitbar.abouturl>
 
 require 'open-uri'
 require 'json'
 
-ENDPOINT = "https://api.etherscan.io/api?module=gastracker&action=gasoracle&apikey="
-APIKEY = ""
+CHAINID = 1 # Ethereum mainnet. See other available chains at https://api.etherscan.io/v2/chainlist and endpoints at https://forms.blockscan.com/public/grid/3E9QiN00NLhCQVibiP3Z-Bpqhmd7zGXsgapEKJupxiI
+ENDPOINT = "ENDPOINT = https://api.etherscan.io/v2/api?chainid=#{CHAINID}&module=gastracker&action=gasoracle&apikey="
+APIKEY = "" # ETHERSCAN API KEY
```

Comment:
While you are updating things, would you consider moving these to be [xbar variables](https://xbarapp.com/docs/2021/03/14/variables-in-xbar.html)?

The `ENDPOINT` could likely remain as a script variable where it needs the chain ID and API key to build it out properly. Also, should it actually be `ENDPOINT = "ENDPOINT = ..."? Feels like a cut / paste error.